Monika After Story/game/script-compliments.rpy

```diff
@@ -530,3 +530,67 @@ label mas_compliment_bestgirl:
     m 5ekbfa "But if you ask me, you made the right choice."
     m 5hubfa "...And I'll be forever grateful that you did~"
     return
+    
+    init 5 python:
+        addEvent(
+            Event(
+                persistent._mas_compliments_database,
+                eventlabel="mas_compliment_you",
+                prompt="... I'm always thinking about you!",
+                unlocked=True
+            ),
+            eventdb=store.mas_compliments.compliment_database
+        )
+
+label mas_compliment_you:
+    if not renpy.seen_label("mas_compliment_you_2"):
+        call mas_compliment_you_2
+    else:
+        call mas_compliment_you_3
+    return
+
+label mas_compliment_you_2:
+    m 1wubso "..."
+    m 1wubfb "[player]..."
```

Comment:
Normally, I don't check expression flow, but I think it might be a bit strange to have these wide eye, happy expressions at the start and then possibly go into the lower, less happy affection paths.

Reply:
My original choice was non-applicable, but I'll find a better option.

Reply:
You could just have your original choice and the expression will be added later if this topic gets approved. The downside is that Travis won't pass you and it might take a bit longer to review.

Reply:
Nah, I found something.